src/test/java/com/powsybl/openloadflow/sensi/ac/AcSensitivityAnalysisTest.java

```diff
@@ -1119,6 +1168,7 @@ void testThreeWindingsTransformerAsFunction() {
     void testThreeWindingsTransformerAsVariable() {
         SensitivityAnalysisParameters sensiParameters = createParameters(false, "b1_vl_0", true);
         sensiParameters.getLoadFlowParameters().setBalanceType(LoadFlowParameters.BalanceType.PROPORTIONAL_TO_GENERATION_P_MAX);
+        sensiParameters.setAngleFlowSensitivityValueThreshold(0.1);
```

Comment:
Are your sure with the modification of the unit test that we lead to an uncovering somewhere else?